whats-new-in-pgroll-0-9-0.mdx

```diff
@@ -0,0 +1,440 @@
+      "create_table": {
+        "name": "events",
+        "columns": [
+          {
+            "name": "id",
+            "type": "serial",
+            "pk": true
+          },
+          {
+            "name": "name",
+            "type": "varchar(255)"
+          },
+          {
+            "name": "start_date",
+            "type": "timestamp"
+          },
+          {
+            "name": "end_date",
+            "type": "timestamp"
+          }
+        ],
+        "constraints": [
+          {
+            "name": "start_before_end",
+            "type": "check",
+            "check": "end_date > start_date"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+### Foreign key constraints
+
+As with the other constraints, simple foreign key constraints can be added at table creation time directly in a column definition:
+
+```json
+{
+  "name": "02_create_orders_table",
+  "operations": [
+    {
+      "create_table": {
+        "name": "orders",
+        "columns": [
+          {
+            "name": "id",
+            "type": "serial",
+            "pk": true
+          },
+          {
+            "name": "user_id",
+            "type": "integer",
+            "references": {
+              "name": "fk_users_id",
+              "table": "users",
+              "column": "id"
+            }
+          },
+          {
+            "name": "quantity",
+            "type": "int"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+For more complex foreign key constraints including different constraint options, use the `constraints` field at table level instead:
+
+```json
+{
+  "name": "02_create_table",
+  "operations": [
+    {
+      "create_table": {
+        "name": "phonebook",
+        "columns": [
+          {
+            "name": "id",
+            "type": "serial",
+            "pk": true
+          },
+          {
+            "name": "provider_id",
+            "type": "serial"
+          },
+          {
+            "name": "name",
+            "type": "varchar(255)"
+          },
+          {
+            "name": "city",
+            "type": "varchar(255)"
+          },
+          {
+            "name": "phone",
+            "type": "varchar(255)"
+          }
+        ],
+        "constraints": [
+          {
+            "name": "provider_fk",
+            "type": "foreign_key",
+            "columns": ["provider_id"],
+            "deferrable": false,
+            "references": {
+              "table": "telephone_providers",
+              "columns": ["id"],
+              "on_delete": "CASCADE",
+              "on_update": "CASCADE",
+              "match_type": "SIMPLE"
+            }
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+### Exclude constraints
+
+It's also possible to specify exclude constraints using the table-level `constraints` field . Examples of how to do so can be found in the [documentation for the create_table operation](https://pgroll.com/docs/v0.9.0/operations/create_table).
```

Comment:
Maybe we could add this example to the post? https://github.com/xataio/pgroll/pull/624#issue-2809177470
The only downside is that it requires the extensions `btree_gist`.

Reply:
Thanks; I've added the example.